### DISCUSSION TITLE
Welcome to BirdNET-Analyzer Discussions!

### DISCUSSION THREAD

**Discussion by kahst** [birdnet-team/BirdNET-Analyzer#231] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by giojacuzzi** [comment]
Hello all! My name is Giordano (Gio) Jacuzzi, I'm studying conservation bioacoustics at the University of Washington as a graduate student. My research uses BirdNET to investigate relationships between forest management in the Pacific Northwest, USA, and bird species occupancy and community composition. I'm looking forward to exchanging ideas and meeting other folks in the community!

On that note, a question I'll be looking into soon is how to retrieve detections for all species in my `species_list` from the analyzer, regardless of confidence (i.e. no minimum confidence threshold). It seems that when I use `min_conf=0.0`, the value is internally overridden to 0.01. I plan to use these data to fit continuous-score occupancy models, which look promising to leverage machine learning vocalization detections at scale without using a threshold cutoff to assign presence/absence labels.

**Comment by JacobGlennAyers** [comment]
Hi there, my name is Jacob Ayers and I am an Electrical Engineering Master's student at UC San Diego. I work with San Diego Zoo researchers that have an ongoing survey in Madre de Dios, Peru. I work to test and develop models to identify species from this abundant region.

Something I am curious about @kahst regarding the Birdnet Ecological Informatics paper. Our team has spent some time working with Xeno-canto data with mixed success with respect to the audio datasets we worth with. One thing we have spent a lot of time looking into is the notion of weak to strong label pipelines. Specifically, techniques similar to those referenced in the Zurich paper where they used median thresholding and binary morphology for foreground-background separation. I am curious if those techniques have ever been tested independently of the larger system?
